data/transactions/logic/opcodes.go

```diff
@@ -184,6 +184,15 @@ func opBranch() OpDetails {
 	return d
 }
 
+func opSwitch() OpDetails {
```

Comment:
I'd prefer another name, simce this seems like the name we would give to the `switch` opcode's evaluation function.